packages/react-templates/src/components/Select/examples/SelectTemplates.md

```diff
@@ -0,0 +1,22 @@
+---
+id: Select
+section: components
+subsection: menus
+template: true
+propComponents: ['SimpleSelect']
```

Comment:
The prop table is not generated.  Does the org PR have to go on first?

Reply:
Yeah I think we have to update the MD routes in org as well for that. cc @nicolethoen